src/views/CountryThreeWProjects/index.tsx

```diff
@@ -0,0 +1,495 @@
+import {
+    useContext,
+    useMemo,
+    useState,
+} from 'react';
+import {
+    isDefined,
+    isNotDefined,
+    listToGroupList,
+    mapToList,
+    unique,
+} from '@togglecorp/fujs';
+import {
+    generatePath,
+    useOutletContext,
+} from 'react-router-dom';
+import {
+    PencilFillIcon,
+    DownloadFillIcon,
+} from '@ifrc-go/icons';
+
+import BlockLoading from '#components/BlockLoading';
+import Button from '#components/Button';
+import Container from '#components/Container';
+import ExpandableContainer from '#components/ExpandableContainer';
+import KeyFigure from '#components/KeyFigure';
+import Link from '#components/Link';
+import PieChart from '#components/PieChart';
+import RouteContext from '#contexts/route';
+import Table from '#components/Table';
+import type { CountryOutletContext } from '#utils/outletContext';
+import type { GoApiResponse } from '#utils/restRequest';
+import useTranslation from '#hooks/useTranslation';
+import { resolveToString } from '#utils/translation';
+import { sumSafe, denormalizeList } from '#utils/common';
+import { type components } from '#generated/types';
+import { useRequest } from '#utils/restRequest';
+import {
+    createActionColumn,
+    createNumberColumn,
+    createStringColumn,
+} from '#components/Table/ColumnShortcuts';
+import {
+    numericIdSelector,
+    numericValueSelector,
+    stringLabelSelector,
+} from '#utils/selectors';
+
+import ProjectActions from './ProjectActions';
+import Map from './Map';
+import Filter, { FilterValue } from './Filters';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+interface LabelValue {
+    label: string;
+    value: number;
+}
+
+type District = NonNullable<GoApiResponse<'/api/v2/district/'>['results']>[number];
+type Project = NonNullable<GoApiResponse<'/api/v2/project/'>['results']>[number];
+
+type ProjectKey = 'reporting_ns' | 'project_districts' | 'programme_type' | 'operation_type' | 'programme_type' | 'primary_sector' | 'secondary_sectors';
+
+// const PROJECT_STATUS_COMPLETED = 2;
+const PROJECT_STATUS_ONGOING = 1 satisfies components['schemas']['Status1d2Enum'];
```

Comment:
Let's assign `components['schemas']['Status1d2Enum']` this type first.
I am not sure if this enum name will change in the figure.
It would be easier to refactor/fix if we have a name